BUILD.bazel

```diff
@@ -731,7 +732,18 @@ cc_binary(
 cc_binary(
     name = "subscription_executor_test",
     testonly = 1,
-    srcs = ["src/ray/gcs/subscription_executor_test.cc"],
+    srcs = ["src/ray/gcs/test/subscription_executor_test.cc"],
+    copts = COPTS,
+    deps = [
+        ":gcs",
+        "@com_google_googletest//:gtest_main",
+    ],
+)
+
+cc_binary(
```

Comment:
Let's avoid adding any more cc_binary scripts that are used for testing. Instead, consider `cc_test` or `sh_test`.

Reply:
Agreed. And I plan to do so in next PR [5515](https://github.com/ray-project/ray/pull/5515)

Reply:
Thanks, that sounds good.